flink-runtime/src/test/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointingStatisticsTest.java

```diff
@@ -18,15 +18,15 @@
 
 package org.apache.flink.runtime.rest.messages.checkpoints;
 
-import org.apache.flink.runtime.checkpoint.CheckpointStatsStatus;
-import org.apache.flink.runtime.jobgraph.JobVertexID;
-import org.apache.flink.runtime.rest.messages.RestResponseMarshallingTestBase;
-
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
+import org.apache.flink.runtime.checkpoint.CheckpointStatsStatus;
```

Comment:
The imports are still not compliant with the CheckStyle format defined for Flink. Please use [Flink's CheckStyle format definition](https://ci.apache.org/projects/flink/flink-docs-stable/flinkDev/ide_setup.html#checkstyle-for-java) to fix the formatting.

Reply:
done